invidious_update.sh

```diff
@@ -287,7 +287,13 @@ else
   echo -e "${RED}${ERROR} Error: Sorry, your OS is not supported.${NC}"
   exit 1;
 fi
-
+# Check if systemd is installed on Devuan
+if [[ $(lsb_release -si) == "Devuan" ]]; then
+  if ( ! $SYSTEM_CMD 2>/dev/null); then
```

Comment:
Er, there is no `systemd` in Devuan. It is freed of systemd ...

Reply:
But Devuan is free of systemd (`start-stop-daemon` does exist), so will it then work?

Reply:
> But Devuan is free of systemd (`start-stop-daemon` does exist), so will it then work?

No, the script is depending on systemd.